back/src/main/resources/db/migration/V1_7__create_tracks.sql

```diff
@@ -0,0 +1,10 @@
+create table if not exists track
+(
+    id bigint not null,
+    name   varchar(100) not null,
+    description   varchar(500),
+    color   varchar(7),
+    open_space_id bigint,
```

Comment:
```suggestion
    id            bigint not null,
    name          varchar(100) not null,
    description   varchar(500),
    color         varchar(7),
    open_space_id bigint,
```

¿es de rompe 🥚 ? puede ser, pero para poder parsearlo mentalmente mejor...